packages/api/src/command/medical/patient/get-consolidated-webhook.ts

```diff
@@ -70,8 +71,9 @@ export async function getConsolidatedWebhook({
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const webhookPayload: any = webhookData.payload;
 
-  const url =
-    webhookPayload.patients?.[0]?.bundle?.entry?.[0]?.resource?.content?.[0]?.attachment?.url;
+  const content = webhookPayload.patients?.[0]?.bundle?.entry?.[0]?.resource?.content;
+  const url = content?.[0]?.attachment?.url;
+  const gzipUrl = content?.[1]?.attachment?.url; // Second attachment is gzip if present
```

Comment:
Can't we have a property on the attachment that says what's the content type? Relying on the index is brittle.